dbml-homepage/docs/docs.md

```diff
@@ -454,7 +454,8 @@ Note: '''
 - Line continuation: `\` backslash
 - Escaping characters:
   - `\`: using double backslash `\\`
-  - `'''`: using `\'''`
+  - `'`: using `\'`
+  - `"`: using `\"`
```

Comment:
I don't remember we ever have this rule, it this behavior newly introduced in this PR or we just didn't document them before?

Reply:
we didn't document this before anh, I think because we explicitly escape `'`, we should mention this rule.